packages/main/src/Dialog.js

```diff
@@ -301,13 +327,8 @@ class Dialog extends Popup {
 	 * Event handlers
 	 */
 	_onDragMouseDown(event) {
-		if (!(this.draggable && this.onDesktop)) {
-			return;
-		}
-
-		// only allow dragging on the header's whitespace
-		if (!event.target.classList.contains("ui5-popup-header-root")
-			&& event.target.getAttribute("slot") !== "header") {
+		// allow dragging only on the header
+		if (!(this._movable && this.draggable && Dialog._isHeader(event.target))) {
```

Comment:
Condition can be simpler:
if (!this._movable || !this.draggable || !Dialog._isHeader(event.target)) { ... }

Reply:
Done